src/components/App/App.css

```diff
@@ -56,3 +56,9 @@ a {
   margin: 1%;
   text-decoration: none;
 }
+
+@media only screen and (max-width: 768px) {
```

Comment:
We should continue to work on adding the hamburger menu so the buttons don't seem so odd, simply stacked one on top of the other